zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/AnormSpanStore.scala

```diff
@@ -321,21 +350,31 @@ class AnormSpanStore(val db: DB,
     }
   }
 
-  override def getSpanNames(_service: String): Future[Seq[String]] = db.inNewThreadWithRecoverableRetry {
-    val service = _service.toLowerCase // service names are always lowercase!
+  override def getSpanNames(_service: Option[String]): Future[Seq[String]] = db.inNewThreadWithRecoverableRetry {
```

Comment:
fwiw I don't think this needs to be an Option unless you need to get a list of all span names for some reason (ex the http api wouldn't call this)